components/Dropdown/Dropdown.module.scss

```diff
@@ -16,13 +16,19 @@
   border-radius: 8px;
   box-shadow: 3px 2px 24px var(--color-black-opacity-1);
   color: var(--color-neutral-80);
+  display: none;
   list-style: none;
   overflow-y: auto;
   padding: 8px !important;
   position: absolute;
   top: 34px;
+  visibility: hidden;
```

Comment:
I consider this line should be removed or we won't see dropdown at all.

Reply:
fixed